.github/workflows/ci.yml

```diff
@@ -48,8 +48,7 @@ jobs:
       matrix:
         try-scenario:
           - ember-lts-3.28
-          - ember-lts-4.4
-          - ember-lts-4.8
+          - ember-lts-4.12
```

Comment:
There's some annoying resolution issues with < 4.11. Makes the most sense to test against 4.12 anyway.